#### Q1. What are the three measures of central tendency?

There are three measures of central tendacy:
1. Mean
2. Median
3. Mode

#### Q2. What is the difference between the mean, median, and mode? How are they used to measure the central tendency of a dataset?

The difference between mean, median and mode are mean is the average of given dataset. 
median is middlemost value of dataset. whereas the mode is most frequently occurring data.

Basically we performs meausure of central tendancy on numerical data. it is try to summarize the data in single value from dataset.

#### Q3. Measure the three measures of central tendency for the given height data:

In [1]:
data = [178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

In [2]:
import numpy as np

In [3]:
np.mean(data)

177.01875

In [4]:
np.median(data)

177.0

In [8]:
from scipy import stats

In [9]:
stats.mode(data)

C:\Users\DELL\AppData\Local\Temp\ipykernel_1832\3267261142.py:1: FutureWarning: Unlike other reduction functions (e.g. `skew`, `kurtosis`), the default behavior of `mode` typically preserves the axis it acts along. In SciPy 1.11.0, this behavior will change: the default value of `keepdims` will become False, the `axis` over which the statistic is taken will be eliminated, and the value None will no longer be accepted. Set `keepdims` to True or False to avoid this warning.
  stats.mode(data)


ModeResult(mode=array([177.]), count=array([3]))

#### Q4. Find the standard deviation for the given data:

In [10]:
data = [178,177,176,177,178.2,178,175,179,180,175,178.9,176.2,177,172.5,178,176.5]

In [11]:
np.std(data)

1.7885814036548633

#### Q5. How are measures of dispersion such as range, variance, and standard deviation used to describe the spread of a dataset? Provide an example.

1. Range is the simplest measure of dispersion. it is simply the total spread in the data, calculated by subtracting the smallest number in the group from the largest number.

    Example: 1,2,3,4,5

    range = (max - min) = 5-1 = 4

2. Variance is subtract mean from each data in the set and square each of them and then add it finally divide them by the total no of value.

    Example: 1, 3, 5, 5, 6, 7, 9, 10.

    mean = (1+ 3+ 5+ 5+ 6+ 7+ 9+ 10)/8 = 46/ 8 = 5.75

    (1 – 5.75), (3 – 5.75), (5 – 5.75), (5 – 5.75), (6 – 5.75), (7 – 5.75), (9 – 5.75), (10 – 5.75)

    = -4.75, -2.75, -0.75, -0.75, 0.25, 1.25, 3.25, 4.25

    Squaring the above values we get, 22.563, 7.563, 0.563, 0.563, 0.063, 1.563, 10.563, 18.063

    22.563 + 7.563 + 0.563 + 0.563 + 0.063 + 1.563 + 10.563 + 18.063
    = 61.504

    n = 8, therefore variance (σ2) = 61.504/ 8 = 7.69

3. Standard deviation is square root of variance is known as std deviation.

   Example: Standard deviation (σ) = 2.77

#### Q6. What is a Venn diagram?

A venn diagram represent mathematical or logical sets pictorially as circle or closed curves within an enclosing rectangle.
Illustrate the relationships between two more set.

#### Q7. For the two given sets A = (2,3,4,5,6,7) & B = (0,2,6,8,10). Find:
i. A ⋂ B
ii. A ⋃ B

A ⋂ B = (2,6)

A ⋃ B = (0,2,3,4,5,6,7,8,10)

#### Q8. What do you understand about skewness in data?

Skewness is how much data set is deviates from its normal distribution.
A larger negative value in the data set means the distribution is negatively skewed, and a larger positive value in the data set means the distribution is positive.

### Q9. If a data is right skewed then what will be the position of median with respect to mean?

Median is less than mean so it available at second position.

#### Q10. Explain the difference between covariance and correlation. How are these measures used in statistical analysis?

Covariance shows you how the two varialbles differ. Whereas correlation shows the how the two variable are related. 

#### Q11. What is the formula for calculating the sample mean? Provide an example calculation for a dataset.

sample mean (s) = xi/n

data = 1,3,4,6,8,9

s = (1+3+4+6+8+8)/6

s = 5

#### Q12. For a normal distribution data what is the relationship between its measure of central tendency?

Measure of central tendancy mean, median and mode will same. in this case mean is best measure of central tendency.

#### Q13. How is covariance different from correlation?

Covariance shows you how the two varialbles differ. Whereas correlation shows the how the two variable are related. 

#### Q14. How do outliers affect measures of central tendency and dispersion? Provide an example.

Outlier are just an number in data set that are vastly larger or smaller  than the other values in data set.
Mean is only measure of central tendancy that is always affected by an outlier.

example: lets say 10 students are there following are scores


In [15]:
score = [0, 88, 90, 92, 94, 95, 95, 96, 97, 99]

In [16]:
np.mean(score)

84.6

the mean is 84.6

lets remove zero from score data set

In [17]:
score = [88, 90, 92, 94, 95, 95, 96, 97, 99]

In [18]:
np.mean(score)

94.0

Now you can see clearly difference.